webui/tests/tests/StepImplementation.ts

```diff
@@ -466,4 +471,22 @@ export default class StepImplementation {
       return logs.length > 0;
     });
   }
+
+  /* Dev */
+  @Step('dev')
+  public async dev() {
+    await this.navigateToTaskList();
```

Comment:
this is some of the attempts I made trying to understand taiko and fix this issue.  I probably should take it out but leave a single command in. or maybe import the contentsof this function from a gitignored file 🤔

Reply:
Yeah, removing all but a single command sounds good.